Modules/TOF/src/CheckHitMap.cxx

```diff
@@ -136,17 +136,17 @@ void CheckHitMap::beautify(std::shared_ptr<MonitorObject> mo, Quality checkResul
     ILOG(Error, Support) << "Cannot beautify MO " << mo->getName() << " which is not of type " << getAcceptedType() << ENDM;
     return;
   }
-  if (mo->getName() == mAcceptedName) {
+  if (1) {
```

Comment:
Shouldn't the `if` be removed in this case?

Reply:
that's true, I could not reapply the formatting online. Will do at the next iteration. Thanks!